### ISSUE TITLE
Support for fade transition

### ISSUE THREAD

**Issue by shaicoleman** [iosscripts/iosslider#16]
It would be nice if it would support the fade effect.

While it doesn't make as much sense for touch devices, it would make it more useful as a general-purpose slider. This could also use CSS3 for the effect.

**Comment by Osifer** [comment]
I totally agree with this one. Would love a to be able to control how quickly it fades on transition.

**Comment by nyquizt** [comment]
indeed!

**Comment by twisk** [comment]
+1

**Comment by rakesh-baikar** [comment]
according to the user interface the slider is good but when its move form first to second its really bad experience for user to see those slider

**Comment by yuchant** [comment]
+1

**Comment by JuTu83** [comment]
+1

**Comment by nguyenDalex** [comment]
+1

**Comment by ashack293** [comment]
+1

**Comment by syuan100** [comment]
+1
